src/GroupManager.php

```diff
@@ -282,17 +284,31 @@ public function refresh() {
   }
 
   /**
-   * Creates default roles for the given group type.
+   * Creates per bundle or per group OG roles.
    *
    * @param string $entity_type_id
    *   The entity type ID of the group for which to create default roles.
-   * @param string $bundle_id
-   *   The bundle ID of the group for which to create default roles.
+   * @param string|null $bundle_id
+   *   (Optional) The bundle ID of the group for which to create default roles.
+   *   When this property is set it means the roles will be "global" roles
+   *   that apply to any group of the bundle ID. Defaults to NULL.
+   * @param string|null $group_id
+   *   (Optional) The group ID of the group for which to create default roles.
+   *   When this property is set it means the roles will be "per group" roles
+   *   and apply only to this specific group ID. Defaults to NULL.
+   *
+   * @see \Drupal\og\GroupManager::createPerBundleRoles()
+   * @see \Drupal\og\GroupManager::createPerGroupRoles()
    */
-  protected function createRoles($entity_type_id, $bundle_id) {
+  protected function createRoles($entity_type_id, $bundle_id = NULL, $group_id = NULL) {
```

Comment:
@pfrenssen I guess it is a bit ugly having to pass either bundle ID or group ID. Which made me thing that maybe we should have two Og Roles bundles: `per_bundle` and `per_group` so the schema will just have those two keys: `group_type` and `reference_id` (bad name) . `reference_id` will be populated either with the bundle ID or the group ID.